ext/pgsql/tests/gh10672.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+GH-10672 (pg_lo_open segfaults in the strict_types mode)
+--EXTENSIONS--
+pgsql
+--SKIPIF--
+<?php
+include("skipif.inc");
+?>
+--FILE--
+<?php
+declare(strict_types=1);
+
+include "config.inc";
+
+function fee(&$a) {}
+$a = ["bar" => "testing"];
+fee($a["bar"]);
+
+$db = pg_connect($conn_str);
+pg_query($db, "DROP TABLE IF EXISTS gh10672");
+pg_query($db, "CREATE TABLE gh8253 (bar text);");
```

Comment:
I've just noticed this problem as well which causes https://github.com/php/php-src/actions/runs/4249761936/jobs/7390212696#step:12:2067